apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -317,7 +292,12 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
   const isBackgroundTransparent = useIsBackgroundTransparent();
 
   const [timeZone, setTimeZone] = useState<string>();
-  const [timeFormat, setTimeFormat] = useState<string>("HH:mm");
+  const [timeFormat, setTimeFormat] = useState<TimeFormat>(detectBrowserTimeFormat);
```

Comment:
I found `h:mma` etc very confusing throughout the code, so I converted in to an enum for way better readability.

In this file we were also using 3 notations: string (`h:mma` etc), numbers (12/24) and `is24Hours` boolean. I got rid of the number variant to have less variants and confusion. Toggle uses the boolean now, all the rest uses string format (with new enum).